prac_06/REFLECTION.md

```diff
@@ -0,0 +1,28 @@
+# CP1404 Practical Reflection
+
+Write short but thoughtful answers to each of the following.  
+Replace each `...` with your meaningful answer.
+
+## What have you changed about your process in this subject based on your personal reflection?
+
+I have actually started using a "TO-DO" app to organise what work
```

Comment:
Nice work !! I do the same using Trello.
I also use an app called AnkiApp to create flip cards and study notes, have found it quite good.